src/components/Letter/LetterThumbnail.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useHistory } from 'react-router-dom';
+import { LetterDataType } from '../../constants/Letters';
+import Text from '../Text/Text';
+
+export interface LetterProps extends Omit<LetterDataType, 'content'> {
+  id: string;
+  thumbnail: string;
+  writer: string;
+}
+
+const Wrapper = styled.div``;
+
+const Writer = styled.div``;
+
+const Thumbnail = styled.div``;
+
+const LetterThumbnail: React.FC<LetterProps> = ({ thumbnail, writer, id }) => {
+  const history = useHistory();
+  return (
+    <Wrapper onClick={() => history.push(`/letter-read/${id}`)}>
+      <Text>
+        <Writer>{writer}</Writer>
+      </Text>

```

Comment:
```suggestion
        <Writer>{writer}</Writer>
```
Writer를 styled 컴포넌트로 Text 감싸게 하면 될 둣